Resources/Prototypes/_Impstation/Entities/Clothing/OuterClothing/coats.yml

```diff
@@ -0,0 +1,14 @@
+- type: entity
+  parent: [ClothingOuterStorageBase, AllowSuitStorageClothing, BaseCommandContraband]
```

Comment:
does parenting this to `ClothingOuterCoatLabCmo` not work?

Reply:
It does not in my attempts to implement it, it causes a fatal error and I can't even start my local host when I do it.

Reply:
what the fuuuuuuck i gotta try it on my localhost